docker-compose.yaml

```diff
@@ -73,7 +74,7 @@ services:
       - ./data/php:/var/www/data
     healthcheck:
       test: curl 127.0.0.1:80/api/v1/json/data/1 | grep -e "data_set_description"
-      start_period: 30s
+      start_period: 2m
```

Comment:
Starting takes very long. I suspect because Elastic Search is running with some compatibility mode, but I wouldn't know. In any case, this allowed me to start the rest-api before. I'm not sure if this period is long enough in all cases, or if it's actually "too long". But the start period ends as soon as the first successful healthcheck is done so it seems there is no harm in setting a longer one.